src/medCore/gui/toolboxes/medAbstractSelectableToolBox.cpp

```diff
@@ -65,3 +65,15 @@ void medAbstractSelectableToolBox::hideEvent(QHideEvent *event)
                    SLOT(updateView()));
     }
 }
+
```

Comment:
Why the calls to ```QApplication::processEvents()```? A developer would not expect these functions to process the events.
Also, ```QApplication::setOverrideCursor``` is a stack based functionality. This means that ```setCursorClassic``` will only work as expected if:
- ```setCursorWait``` was called before
- Since the last call to ```setCursorWait```, no other code has overridden the cursor without changing it back.

For the reasons stated above I think that these two functions are not needed, and it is better to let the developer call ```QApplication::setOverrideCursor``` and ```QApplication::restoreOverrideCursor``` directly.